src/main/java/com/google/devtools/build/lib/rules/apple/AppleCommandLineOptions.java

```diff
@@ -38,17 +38,6 @@
 
 /** Command-line options for building for Apple platforms. */
 public class AppleCommandLineOptions extends FragmentOptions {
-  @Option(
-      name = "experimental_apple_mandatory_minimum_version",
```

Comment:
I can graveyard the flag if folks would prefer but it's been dead for long enough probably